com.github.matfantinel.reminduck.json

```diff
@@ -0,0 +1,70 @@
+{
+    "app-id": "com.github.matfantinel.reminduck",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "command": "com.github.matfantinel.reminduck",
+    "finish-args": [
+       "--share=ipc",
+       "--share=network",
+       "--socket=fallback-x11",
+       "--socket=wayland",
+       "--talk-name=org.gtk.vfs.*",
+       "--filesystem=home"
+    ],
+    "cleanup": [
+       "/include",
+       "/lib/pkgconfig",
+       "/man",
+       "/share/doc",
+       "/share/gtk-doc",
+       "/share/man",
+       "/share/pkgconfig",
+       "/share/vala",
+       "*.la",
+       "*.a"
+    ],
+    "modules": [
+       {
+          "name": "libyaml",
+          "sources": [
+              {
+                  "type": "git",
+                  "url": "https://github.com/yaml/libyaml.git",
+                  "tag": "0.2.2",
+                  "commit": "690a781ef6af70ce6749d6e2be91743345123998"
+              }
+          ]
+       },
+       {
+          "name": "appstream",
+          "buildsystem": "meson",
+          "config-opts": [
+              "-Dstemming=false"
+          ],
+          "sources": [
+              {
+                  "type": "git",
+                  "url": "https://github.com/ximion/appstream.git",
+                  "tag": "v0.12.9",
+                  "commit": "d77cc0f331ff5d3bd43be104106a94ed201c62ec"
+              }
+          ]
```

Comment:
Include lmdb as a submodule of appstream https://github.com/flathub/io.github.NhekoReborn.Nheko/blob/master/io.github.NhekoReborn.Nheko.json#L28